addons/portal_rating/views/rating_templates.xml

```diff
@@ -88,7 +88,7 @@
                         <t t-else="">Add Review</t>
                     </span>
                 </button>
-                <div class="o_rating_popup_composer_modal"/>
+                <div class="o_rating_popup_composer_modal bg-white"/>
```

Comment:
Actually, I think it will be better to add the class `bg-white` on the `modal-content` div of `portal_rating.PopupComposer` because the `modal-content` div has its own background color and will not use the background color set by `bg-white`.

By adding the class on the `modal-content` div, we will set the text color and the background color of the modal. It will guarantee that there will be a good contrast no matter which theme the user uses :smiley: